CurrencyConversor/CurrencyConversor/Model/CalculatorManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CalculatorManager.swift
+//  CurrencyConversor
+//
+//  Created by Leidy Luna on 29/08/23.
+//
+
+import Foundation
+
+
+protocol CalculadorManagerDelegate {
+    func didUpdateResult(_ currencyResult: String)
+    func didFailWithError(error: Error)
+}
+
+struct CalculatorManager {
```

Comment:
A struct should not be used for a manager

Reply:
Why no, i used because in the bootcamp, we used for all Manager  por example struct WeatherManager {
And what do you thing i need to use